cdap-proto/src/main/java/co/cask/cdap/proto/Id.java

```diff
@@ -17,11 +17,14 @@
 package co.cask.cdap.proto;
 
 import com.google.common.base.CharMatcher;
+import com.google.common.base.Charsets;
 import com.google.common.base.Objects;
 import com.google.common.base.Preconditions;
 import com.google.common.base.Splitter;
 import com.google.common.collect.Iterables;
 
+import javax.ws.rs.HEAD;
```

Comment:
This came in during resolving merge conflicts; IDE didn't remove unused for some reason ;s